main/autogen/autogen.cc

```diff
@@ -674,4 +687,34 @@ void ParsedFile::classlist(core::Context ctx, vector<string> &out) {
     }
 }
 
+void ParsedFile::subclasses(core::Context ctx, map<string, set<string>> &out) {
+    auto nameToString = [&](const core::NameRef &nm) -> string { return nm.data(ctx)->show(ctx); };
+
+    for (const Reference &ref : refs) {
+        // Ignore files in dirs we don't care about
+        if (path.find("test/") == 0 || path.find("scripts/") == 0 || path.find("lib/identification/scripts/") == 0 ||
```

Comment:
In the autoloader PR I'm working on I also have some uglier stripe-specific config. Let's coordinate on how to supply that.